packages/donor/src/screens/extendedSignup/PersonalDetails.tsx

```diff
@@ -43,8 +43,17 @@ export default function PersonalDetails(props: PersonalDetailsProps) {
     undefined
   );
 
+  const isPhoneNumberAllNumbers = /^[0-9]*$/.test(phone);
   const phoneValidator = /^05(?!6)\d{8}$/;
   const isValidPhone = phoneValidator.test(phone);
+
+  let phoneMessage = undefined;
```

Comment:
errorMessage

Reply:
didn't understand (it stayed like before - error message or undefined if no error)

Reply:
I meant change the variable name to `errorMessage`, not `phoneMessage`

Reply:
Oh, ok.